mzLib/Chromatography/RetentionTimePrediction/CZE/CZERetentionTimePredictor.cs

```diff
@@ -0,0 +1,105 @@
+using Chromatography.RetentionTimePrediction.Util;
+
+namespace Chromatography.RetentionTimePrediction.CZE;
+
+/// <summary>
+/// CZE-based retention time predictor using electrophoretic mobility calculations.
+/// Predicts migration times for peptides in capillary zone electrophoresis.
+/// 
+/// Based on: Krokhin et al., Anal Chem. 2017 Feb 7;89(3):2000-2008.
+/// "Predicting Electrophoretic Mobility of Tryptic Peptides for High-Throughput CZE-MS Analysis"
+/// </summary>
+public class CZERetentionTimePredictor : RetentionTimePredictor
+{
+    private readonly double _columnLengthMeters;
+    private readonly double _voltsPerMeter;
+
+    public override string PredictorName => "CZE";
+    public override SeparationType SeparationType => SeparationType.CZE;
+ 
+    /// <summary>
+    /// Initializes a new CZE predictor with default parameters (100cm column, 300kV/m)
+    /// </summary>
+    public CZERetentionTimePredictor(
+        IncompatibleModHandlingMode modHandlingMode = IncompatibleModHandlingMode.UsePrimarySequence)
+        : this(modHandlingMode, columnLengthMeters: 1.0, voltsPerMeter: 300000)
+    {
+    }
+
```

Comment:
Can probably just get rid of this guy and add default arguments to the parameters in the constructor on line 35